docs/developer/ceed.md

```diff
@@ -0,0 +1,206 @@
+
+# RDycore SWE physics with CEED
+
+Here we describe the RDycore's implemetation of 2D shallow water equation (SWE) that uses [libCEED](https://github.com/CEED/libCEED). The combination of PETSc and libCEED provides RDDycore algorithimic and hardware protability.
```

Comment:
implemetation -> implementation

provides RDDycore algorithimic and hardware protability -> provides RDycore with a combination of algorithmic transparency and performance portability.